.gitignore

```diff
@@ -11,4 +11,7 @@ deployment/static/
 deployment/media/
 deployment/volumes/
 celery.log
-django_project/celerybeat-schedule
\ No newline at end of file
+django_project/celerybeat-schedule
+
+.env
+docker-compose.override.yml
```

Comment:
Allow developer local specific configuration overrides from `.env` file and `docker-compose.override.yml`